x-pack/plugins/maps/public/classes/util/can_skip_fetch.ts

```diff
@@ -69,6 +69,13 @@ export async function canSkipSourceUpdate({
     return false;
   }
 
+  if (
```

Comment:
Can you move this if statement below `const prevMeta = prevDataRequest.getMeta();` and use prevMeta instead of `prevDataRequest?.getMeta()`